src/topology/category/Compactum.lean

```diff
@@ -100,14 +100,14 @@ instance : has_limits Compactum := has_limits_of_has_limits_creates_limits forge
 def str (X : Compactum) : ultrafilter X → X := X.a
 
 /-- The monadic join. -/
-def join (X : Compactum) : ultrafilter (ultrafilter X) → ultrafilter X := (μ_ β).app _
+def join (X : Compactum) : ultrafilter (ultrafilter X) → ultrafilter X := β .μ.app _
```

Comment:
Similarly, the space in `β .μ` seems unnecessary.

Reply:
As above, unfortunately it's not.